Files.OpenDialog/CustomOpenDialog/CustomOpenDialog.vcxproj

```diff
@@ -30,43 +30,43 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{A2FF3F3B-8EBC-4108-B99D-1476B7876656}</ProjectGuid>
     <Keyword>AtlProj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
Could we keep changes to this file (if necessary) to a separate PR?

Reply:
Would be nice if GitHub had the option to split a PR 🤔

Reply:
@d2dyno1 this is needed to compile on VS2022 I assume?

Reply:
VS adds them automatically, these don't prevent from building on VS2022.

Reply:
Asking as I'm stuck on VS2019 and this changes do prevent _me_ from building =P

Reply:
Why won't you upgrade to VS2022? It builds fine for me